resources/install.rb

```diff
@@ -7,6 +7,8 @@
 property :influxdb_key, kind_of: String, default: 'https://repos.influxdata.com/influxdb.key'
 property :install_version, [String, nil], default: nil
 property :install_type, String, default: 'package'
+property :package_name, String, name_property: true
+property :checksum, String, default: node['influxdb']['shasums'][node[:platform_family]]
```

Comment:
Foodcritic is complaining about this line ref: http://www.foodcritic.io/#FC019

It should be

```ruby
property :checksum, String, default: node['influxdb']['shasums'][node['platform_family']]
```

Reply:
absolutely